From 1dc0e2c75df089cdf1acb78192381f40088bba01 Mon Sep 17 00:00:00 2001
From: Marty Jones <mj8263788@gmail.com>
Date: Thu, 29 Sep 2022 18:44:42 -0400
Subject: [PATCH 17/95] drivers: crypto-rockchip fix openwrt build

Signed-off-by: Marty Jones <mj8263788@gmail.com>
---
 drivers/crypto/rockchip/rk3288_crypto.c | 12 ------------
 1 file changed, 12 deletions(-)

--- a/drivers/crypto/rockchip/rk3288_crypto.c
+++ b/drivers/crypto/rockchip/rk3288_crypto.c
@@ -31,18 +31,6 @@ static struct rockchip_ip rocklist = {
 	.lock = __SPIN_LOCK_UNLOCKED(rocklist.lock),
 };
 
-struct rk_crypto_info *get_rk_crypto(void)
-{
-	struct rk_crypto_info *first;
-
-	spin_lock(&rocklist.lock);
-	first = list_first_entry_or_null(&rocklist.dev_list,
-					 struct rk_crypto_info, list);
-	list_rotate_left(&rocklist.dev_list);
-	spin_unlock(&rocklist.lock);
-	return first;
-}
-
 static const struct rk_variant rk3288_variant = {
 	.num_clks = 4,
 	.rkclks = {
