Index: hostapd-2023-06-22-599d00be/src/drivers/driver_nl80211.c
===================================================================
--- hostapd-2023-06-22-599d00be.orig/src/drivers/driver_nl80211.c
+++ hostapd-2023-06-22-599d00be/src/drivers/driver_nl80211.c
@@ -7686,7 +7686,7 @@ static int get_sta_handler(struct nl_msg
 		[NL80211_STA_INFO_BEACON_SIGNAL_AVG] = { .type = NLA_U8},
 		[NL80211_STA_INFO_RX_DURATION] = { .type = NLA_U64 },
 		[NL80211_STA_INFO_ACK_SIGNAL] = { .type = NLA_U8 },
-		[NL80211_STA_INFO_ACK_SIGNAL_AVG] = { .type = NLA_S8 },
+		[NL80211_STA_INFO_ACK_SIGNAL_AVG] = { .type = NLA_U8 },
 		[NL80211_STA_INFO_RX_MPDUS] = { .type = NLA_U32 },
 		[NL80211_STA_INFO_FCS_ERROR_COUNT] = { .type = NLA_U32 },
 		[NL80211_STA_INFO_TX_DURATION] = { .type = NLA_U64 },
@@ -7792,7 +7792,7 @@ static int get_sta_handler(struct nl_msg
 	}
 	if (stats[NL80211_STA_INFO_ACK_SIGNAL_AVG])
 		data->avg_ack_signal =
-			nla_get_s8(stats[NL80211_STA_INFO_ACK_SIGNAL_AVG]);
+			nla_get_u8(stats[NL80211_STA_INFO_ACK_SIGNAL_AVG]);
 	if (stats[NL80211_STA_INFO_RX_MPDUS])
 		data->rx_mpdus = nla_get_u32(stats[NL80211_STA_INFO_RX_MPDUS]);
 	if (stats[NL80211_STA_INFO_FCS_ERROR_COUNT])
  