--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -157,6 +158,7 @@ dtb-$(CONFIG_ROCKCHIP_RK3399) += \
 	rk3399-nanopi-m4b.dtb \
 	rk3399-nanopi-neo4.dtb \
 	rk3399-nanopi-r4s.dtb \
+	rk3399-nanopi-r4se.dtb \
 	rk3399-orangepi.dtb \
 	rk3399-pinebook-pro.dtb \
 	rk3399-pinephone-pro.dtb \

 dtb-$(CONFIG_ROCKCHIP_RK3588) += \
@@ -197,7 +203,9 @@ dtb-$(CONFIG_ROCKCHIP_RK3588) += \
 	rk3588-orangepi-5-plus.dtb \
 	rk3588-quartzpro64.dtb \
 	rk3588s-rock-5a.dtb \
-	rk3588-rock-5b.dtb
+	rk3588-rock-5b.dtb \
+	rk3588s-nanopi-r6c.dtb \
+	rk3588s-nanopi-r6s.dtb
 
 dtb-$(CONFIG_ROCKCHIP_RV1108) += \
 	rv1108-elgin-r1.dtb \
